### ISSUE TITLE
How to change application name

### ISSUE THREAD

**Issue by Scheiderichv094** [OfficeDev/microsoft-teams-apps-company-communicator#957]
I am wanting to change the Company Communicator name and want it to reflect in Azure and MS Teams. What are all the steps I need to take in order to change the name successfully for both the user and Author app?

**Comment by gsv022** [comment]
Hi @Scheiderichv094 ,
Thanks for raising the query. Please follow the below steps.
To update the author app, user app icons and name 

- Login to Azure Portal with Global Admin Credentials (portal.azure.com)
- Go to Resource Groups and Under your subscription select the resource group where the app is deployed.
-  Click on Azure Bot(user) --> under settings click bot profile-->Change the icon and display name as per the organizational need.
<img width="749" alt="image" src="https://user-images.githubusercontent.com/98143492/213981428-e5af8e1e-8623-4731-b413-8cc62cc3c475.png">

- And update the icons name in the manifest files for both author and user
<img width="280" alt="image" src="https://user-images.githubusercontent.com/98143492/213983428-68678fb4-03ae-4a95-933e-0d8e4b4f802d.png">


**Note**:- 
- There might be chance that desktop app will get cached. However, it can be validated by checking in the browser version of the app.
- In the ‘Icon’ field, it is recommended to upload the size **192x192** 

And if company communicator is of 5.x version, needs the header logo and header text needs to changed then follow this 
[link](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/v5-migration-guide#2-validate-and-update-the-config-settings-and-sync-to-the-lastest-code-)
<img width="715" alt="image" src="https://user-images.githubusercontent.com/98143492/213982701-b95cb3ca-524a-4b4f-bcf0-e227f2c7f87f.png">

**Comment by Scheiderichv094** [comment]
Thank you for this information! If we use a new manifest, do we also need to make sure to include the bot ID and app domain? Also, if it caches on the desktop after this is done, does the user just need to clear the cache from Teams or sign out and sign back in?

**Comment by gsv022** [comment]
Yes, for new manifest including botId and app domain is mandatory. Once the modifications are done wait for sometime and if changes are not reflecting in desktop teams, try opening in browser and see the changes. 

Please let us know if you have any queries

**Comment by Scheiderichv094** [comment]
Once this is all complete and synced, the users in our organization should see the logo and name change within MS Teams? Do you know of any errors or issues other users ran into while changing the logo and name?

**Comment by gsv022** [comment]
Yes, once the modifications are done change should reflect.

**Comment by Scheiderichv094** [comment]
If it does not change for the user in Teams, do you suggest clearing the Teams cache for them or having them sign out then sign back into Teams?

**Comment by gsv022** [comment]
If the changes are not reflecting in desktop teams please try opening the teams in browser and check for the latest changes.

**Comment by ndl7** [comment]
> If the changes are not reflecting in desktop teams please try opening the teams in browser and check for the latest changes.

We changed the name 4 days ago, it does reflect in web version of Teams. Still not coming up in the desktop version unless cache is cleared. We got tens of thousands of users so asking them to clear their Teams cache is not really an option. Is there any way to force refresh for everyone using desktop version? Thanks

**Comment by Scheiderichv094** [comment]
I did find this article,

https://techcommunity.microsoft.com/t5/itops-talk-blog/powershell-basics-how-to-delete-microsoft-teams-cache-for-all/ba-p/1519118

**Comment by ndl7** [comment]
> I did find this article,
> 
> https://techcommunity.microsoft.com/t5/itops-talk-blog/powershell-basics-how-to-delete-microsoft-teams-cache-for-all/ba-p/1519118

Sorry for not making this 'clear', but these tens of thousands of users are not using the same computer, so that article is not really helpful. Is there any way how to refresh this change somewhere centrally?

**Comment by Scheiderichv094** [comment]
Do you have an application such as Altiris or WorkSpaceOne where you can push software or script to your whole company? That is what we use to push that script to the whole org to clear the users cache
